lib/drivers/veres/index.js

```diff
@@ -21,6 +25,78 @@ const _keyTypes = {
   rsa: 'RsaVerificationKey2018'
 };
 
+function driver({mode, hostname}) {
+  // TODO: Support custom httpsAgent?
+  const httpsAgent = new https.Agent({rejectUnauthorized: mode !== 'dev'});
+
+  return v1.driver({mode, hostname, httpsAgent});
+}
+
+/**
+ * @param {string} [type='veres'] - DID type / method.
+ * @param {string} [mode='test'] - Ledger mode (test/live/dev).
+ *
+ * @returns {{pending: FlexDocStore, registered: FlexDocStore}}
+ */
+function didStoreFor({type, mode}) {
+  return {
+    pending: didStore({type, mode, status: PENDING}),
+    registered: didStore({type, mode, status: REGISTERED})
+  };
+}
+
+function displayDidDoc({options, didDocument, status, meta}) {
+  // TODO: Add registration status and local metadata to result.
+  logger._log(options, JSON.stringify(didDocument, null, 2));
+  logger._log(options, 'STATUS:', status);
+  logger._log(options, 'METADATA:', JSON.stringify(meta, null, 2));
+}
+
+/**
+ * @param {string} did - DID url.
+ * @param {VeresOne} veresDriver
+ * @param {string} [mode='test'] - Ledger mode (test/live/dev), if applicable.
+ * @param {string} [type='veres'] - DID type / method.
+ *
+ * @returns {Promise<{meta: object, didDocument: object, status: string}>}
+ */
+async function loadDidDocument({did, veresDriver, mode, type}) {
+  // Load metadata (to see if it's been published, etc)
+  const meta = (await metaStore().get(did)) || {};
+
+  let status, didDocument;
+
+  if(meta.published) {
+    status = REGISTERED;
+    didDocument = await veresDriver.get({did});
+  } else {
+    status = PENDING;
+    const doc = await didStore({type, mode, status: PENDING}).get(did);
+    didDocument = new VeresOneDidDoc({doc});
+  }
+
+  await didDocument.importKeys(await keyStore().get(did));
+
+  return {status, didDocument, meta};
+}
+
+/**
+ * @param {VeresOneDidDoc} didDocument
+ * @param {string} status - registered/pending.
+ * @param {string} [type='veres'] - DID type / method.
+ * @param {string} [mode='test'] - Ledger mode (test/live/dev).
+ * @param {VeresOne} veresDriver
+ * @returns {Promise<void>}
+ */
+async function saveDidDocument({didDocument, status, type, mode, veresDriver}) {
+  if(status === PENDING) {
+    const did = didDocument.id;
+    await didStore({type, mode, status: PENDING}).put(did, didDocument);
+  } else { // REGISTERED
```

Comment:
since there's no further processing, just return early on L94 and remove else.

Reply:
Changed, thanks.